internal/blueprint/blueprint_test.go

```diff
@@ -0,0 +1,32 @@
+package blueprint
+
+import "testing"
+
+func TestDeepCopy(t *testing.T) {
+	bpOrig := Blueprint{
+		Name:        "deepcopy-test",
+		Description: "Testing DeepCopy function",
+		Version:     "0.0.1",
+		Packages: []Package{
+			{Name: "dep-package1", Version: "*"}},
+		Modules: []Package{
+			{Name: "dep-package2", Version: "*"}},
+	}
+
+	bpCopy, err := bpOrig.DeepCopy()
```

Comment:
I think we should have an equality test of the original and the copy here.

Reply:
Good idea, I added cmp.Diff so we can see what went wrong.